tests/boardfarm_plugins/boardfarm_prplmesh/tests/client_association_link_metrics.py

```diff
@@ -71,3 +61,4 @@ def teardown_class(cls):
             # If AttributeError was raised - we are dealing with dummy devices.
             # We don't have to additionaly send Ctrl+C for dummy devices.
             pass
+        test.dev.wifi.disable_wifi()
```

Comment:
Here as well, a better naming IMO is `test.dev.wifi.disable()`

Reply:
This is build-in boardfarm method, so we may create a wrapper.